code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -726,6 +726,8 @@
 // called from mob/living/carbon/human/Crossed()
 // when mulebot is in the same loc
 /mob/living/simple_animal/bot/mulebot/proc/RunOver(mob/living/carbon/human/H)
+	if(!wires.is_cut(WIRE_AVOIDANCE))
+		return ..()
```

Comment:
```suggestion
		return
```
`..()` calls the parent proc, but this proc has no parent
you can know that because it has `/proc/` in its name
redefined procs don't have that

Reply:
Ohh, I see. Thanks!